https://github.com/pytest-dev/pytest/issues/12186

Please note it is currently impossible to uninstall typeguard because
package/pkg transitionally depends on it.  The dependency chain is:

package/pkg -> cherrypy -> jaraco-collections -> jaraco-text -> inflect -> typeguard

--- pytest_ignore_flaky-2.2.1/tests/test_functional.py.orig
+++ pytest_ignore_flaky-2.2.1/tests/test_functional.py
@@ -53,6 +53,7 @@
 TEST_SAMPLE_SUCCEED = files().joinpath('sample succeed.py').read_text(encoding='utf-8')
 
 
+@pytest.mark.skip(reason="fails with typeguard installed")
 def test_success_flaky(testdir, capsys):
     test = testdir.makepyfile(TEST_SAMPLE_SUCCEED)
     rec = testdir.inline_run('--ignore-flaky', test)
